mixins/form.jsx

```diff
@@ -193,7 +193,7 @@ module.exports = {
     if (errorType === "card_error" && cardError) {
       form.error(cardError.field, cardError.message);
     } else {
-      form.error("other", this.getIntlMessage('try_again_later'));
+      form.error("other", this.getIntlMessage('try_again_later') + " [" + errorType + "]");
```

Comment:
So, we are going to get this

`Something seems to have gone wrong. Please try again later ["api_connection_error"]`.

This is fine, but if we are hoping if any user (donors) to report back an error then surely we need a better message here. (I won't send a screenshot if it happened to me when trying to donate somewhere LOL).